src/engine/SCons/SConf.py

```diff
@@ -1067,7 +1067,7 @@ def CheckLibWithHeader(context, libs, header, language,
     """
     prog_prefix, dummy = \
                  createIncludesFromHeaders(header, 0)
-    if libs == []:
+    if not libs:
```

Comment:
Not sure it matters, but this would be true of libs = False or 0 or other values and not just for []?